larkspur/larkspur.py

```diff
@@ -16,7 +17,13 @@ def deserialize_hm(hm):
     return out
 
 
-def make_hashes(num_slices, num_bits):
+def make_hashes(num_slices: int, num_bits: Any):
```

Comment:
You can confirm, but I think this needs to be an `int`

```suggestion
def make_hashes(num_slices: int, num_bits: int):
```